src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpInitializeStaticFieldsInline.Fixer.cs

```diff
@@ -4,15 +4,14 @@
 using Microsoft.NetCore.Analyzers.Runtime;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
-using Microsoft.CodeAnalysis.CSharp;
 
 namespace Microsoft.NetCore.CSharp.Analyzers.Runtime
 {
     /// <summary>
     /// CA2207: Initialize value type static fields inline
     /// </summary>
     [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
-    public class CSharpInitializeStaticFieldsInlineFixer : InitializeStaticFieldsInlineFixer<SyntaxKind>
+    public class CSharpInitializeStaticFieldsInlineFixer : InitializeStaticFieldsInlineFixer
     {
     }
```

Comment:
I think this can be completely deleted and have only one fixer in `Core` layer.

Reply:
Agreed.